lua/telekasten.lua

```diff
@@ -203,6 +199,14 @@ local function escape(s)
     return string.gsub(s, "[%%%]%^%-$().[*+?]", "%%%1")
 end
 
+local function make_absolute_path(path)
+    local ret = path
+    if not (Path:new(path):is_absolute()) and path ~= nil then
+        ret = M.Cfg.home .. "/" .. path
+    end
+    return ret
+end
+
```

Comment:
OK, this looked suspicious since it does not deal with `daily/`, etc. But then I realized, it's not for resolving links. Mayyyybe I'll rename it to something like `make_absolute_path_from_config_path()` just to make sure nobody gets tempted to use it for resolving links which, to some extent, are also relative paths.

Reply:
Make sense. Maybe `make_config_path_absolute()`to keep it somewhat shorter.

Reply:
Ok done